config/authelia/configuration.yml

```diff
@@ -0,0 +1,1423 @@
+      #  - '^appgroup2-.*\.example\.com$'
+    #   policy: 'one_factor'
+    # - domain_regex: '^.*\.example\.com$'
+    #   policy: 'two_factor'
+
+    # - domain: 'secure.example.com'
+    #   policy: 'one_factor'
+    ## Network based rule, if not provided any network matches.
+    #   networks:
+        # - 'internal'
+        # - 'VPN'
+        # - '192.168.1.0/24'
+        # - '10.0.0.1'
+
+    # - domain:
+        # - 'secure.example.com'
+        # - 'private.example.com'
+    #   policy: 'two_factor'
+
+    # - domain: 'singlefactor.example.com'
+    #   policy: 'one_factor'
+
+    ## Rules applied to 'admins' group
+    # - domain: 'mx2.mail.example.com'
+    #   subject: 'group:admins'
+    #   policy: 'deny'
+
+    # - domain: '*.example.com'
+    #   subject:
+        # - 'group:admins'
+        # - 'group:moderators'
+    #   policy: 'two_factor'
+
+    ## Rules applied to 'dev' group
+    # - domain: 'dev.example.com'
+    #   resources:
+        # - '^/groups/dev/.*$'
+    #   subject: 'group:dev'
+    #   policy: 'two_factor'
+
+    ## Rules applied to user 'john'
+    # - domain: 'dev.example.com'
+    #   resources:
+        # - '^/users/john/.*$'
+    #   subject: 'user:john'
+    #   policy: 'two_factor'
+
+    ## Rules applied to user 'harry'
+    # - domain: 'dev.example.com'
+    #   resources:
+        # - '^/users/harry/.*$'
+    #   subject: 'user:harry'
+    #   policy: 'two_factor'
+
+    ## Rules applied to user 'bob'
+    # - domain: '*.mail.example.com'
+    #   subject: 'user:bob'
+    #   policy: 'two_factor'
+    # - domain: 'dev.example.com'
+    #   resources:
+    #     - '^/users/bob/.*$'
+    #   subject: 'user:bob'
+    #   policy: 'two_factor'
+
+##
+## Session Provider Configuration
+##
+## The session cookies identify the user once logged in.
+## The available providers are: `memory`, `redis`. Memory is the provider unless redis is defined.
+session:
+  ## The secret to encrypt the session data. This is only used with Redis / Redis Sentinel.
+  ## Secret can also be set using a secret: https://www.authelia.com/c/secrets
+  # secret: 'insecure_session_secret'
+
+  ## Cookies configures the list of allowed cookie domains for sessions to be created on.
+  ## Undefined values will default to the values below.
+  cookies:
+    -
+      ## The name of the session cookie.
+      #name: 'authelia_session'
+
+      ## The domain to protect.
+      ## Note: the Authelia portal must also be in that domain.
+      domain: '{{ mustEnv "APP_DOMAIN" }}'
+
+      ## Required. The fully qualified URI of the portal to redirect users to on proxies that support redirections.
+      ## Rules:
+      ##   - MUST use the secure scheme 'https://'
+      ##   - The above 'domain' option MUST either:
+      ##      - Match the host portion of this URI.
+      ##      - Match the suffix of the host portion when prefixed with '.'.
+      authelia_url: 'https://auth.{{ mustEnv "APP_DOMAIN" }}'
```

Comment:
Think this will need updating to the new Authelia address `https://{{ mustEnv "APP_DOMAIN"}}/authelia`

Reply:
Ah, thanks, updated.